quantum/snippets.js

```diff
@@ -56,10 +56,23 @@ counts = result.get_counts()
 print(counts)
 `;
 
-const IBMQ_SYSTEM_DEFAULT =
+const IBMQ_SYSTEM_QASM =
 `from qiskit.providers.ibmq import least_busy
 provider = IBMQ.enable_account('%s')
-backends = provider.backends(filters=lambda x: x.configuration().n_qubits >= %s)
+backend_service = provider.get_backend('ibmq_qasm_simulator')
+`;
+
+const IBMQ_SYSTEM_DEFAULT_SIMUL =
+`from qiskit.providers.ibmq import least_busy
+provider = IBMQ.enable_account('%s')
+backends = provider.backends(filters=lambda x: (x.configuration().n_qubits >= %s and x.configuration().simulator == True))
+backend_service = least_busy(backends)
+`;
+
+const IBMQ_SYSTEM_DEFAULT_QUBIT =
+`from qiskit.providers.ibmq import least_busy
+provider = IBMQ.enable_account('%s')
+backends = provider.backends(filters=lambda x: (x.configuration().n_qubits >= %s and x.configuration().simulator == False))
 backend_service = least_busy(backends)
 `;
```

Comment:
Given that these code blocks are practically the same, can we extract this into a single string and use string formatting for setting the simulator as true/false? So it would be something like:
```suggestion
const IBMQ_SYSTEM_DEFAULT =
`from qiskit.providers.ibmq import least_busy
provider = IBMQ.enable_account('%s')
backends = provider.backends(filters=lambda x: (x.configuration().n_qubits >= %s and x.configuration().simulator == %s))
backend_service = least_busy(backends)
`;
```

And we format the string as:
```node
script += util.format(snippets.IBMQ_SYSTEM_DEFAULT, node.apiToken, node.qubits.length, 'True');
// OR
script += util.format(snippets.IBMQ_SYSTEM_DEFAULT, node.apiToken, node.qubits.length, 'False');
```

Reply:
Good suggestion, Louis. I'll implement it.